marlowe-playground-client/static/css/modal.scss

```diff
@@ -17,7 +17,7 @@ without doing it as an Effect in a mainframe action
   /* Stay in place */
   position: fixed;
   /* Sit on top */
-  z-index: 1;
+  z-index: 100; // Blockly toolbar has z-index of 70, so we need this be bigger than that
```

Comment:
It would be good to find some way of keeping track of z-indexes across the site to avoid this sort of problem. 🤔